### DISCUSSION TITLE
Complex cEOS-lab topology : 2 spines , 4 Leafs, 2 Multihomed Linux Hosts : Get Errors

### DISCUSSION THREAD

**Discussion by dharmbhai** [srl-labs/containerlab#358] [Q&A] [answered]
# topology documentation: http://containerlab.srlinux.dev/lab-examples/min-clos/
# topology documentation: 2 spines and 4 leafs with 2 hosts
name: ceos-clos01

topology:
  kinds:
    ceos:
      image: ceos
    linux:
      image: ghcr.io/hellt/network-multitool
  nodes:
    leaf1:
      kind: ceos
    leaf2:
      kind: ceos
    leaf3:
      kind: ceos
    leaf4:
      kind: ceos
    spine1:
      kind: ceos
    spine2:
      kind: ceos
    client1:
      kind: linux
    client2:
      kind: linux

  links:
    - endpoints: ["leaf1:e2", "spine1:e2"]
    - endpoints: ["leaf1:e3", "spine2:e2"]
    - endpoints: ["leaf2:e2", "spine1:e3"]
    - endpoints: ["leaf2:e3", "spine2:e3"]
    - endpoints: ["leaf3:e2", "spine1:e4"]
    - endpoints: ["leaf3:e3", "spine2:e4"]
    - endpoints: ["leaf4:e2", "spine1:e5"]
    - endpoints: ["leaf4:e3", "spine2:e5"]
    - endpoints: ["leaf1:e10", "leaf2:e10"]
    - endpoints: ["leaf1:e11", "leaf2:e11"]
    - endpoints: ["leaf3:e10", "leaf4:e10"]
    - endpoints: ["leaf3:e11", "leaf4:e11"]
    - endpoints: ["client1:eth1", "leaf1:e4"]
    - endpoints: ["client1:eth2", "leaf2:e4"]
    - endpoints: ["client2:eth1", "leaf3:e4"]
    - endpoints: ["client2:eth2", "leaf4:e4"]

###################Output ################
dharmesh@dharmesh-ubuntu:~/clab-quickstart/clab-ceos-clos01$ sudo containerlab deploy --reconfigure --topo ceos-clos01.clab.yml
INFO[0000] Parsing & checking topology file: ceos-clos01.clab.yml
INFO[0000] Destroying container lab: ceos-clos01
INFO[0000] Removing container entries from /etc/hosts file
INFO[0000] Deleting docker network 'clab'...
INFO[0000] Removing /home/dharmesh/clab-quickstart/clab-ceos-clos01/clab-ceos-clos01 directory...
INFO[0000] Creating lab directory: /home/dharmesh/clab-quickstart/clab-ceos-clos01/clab-ceos-clos01
INFO[0000] Creating docker network: Name='clab', IPv4Subnet='172.20.20.0/24', IPv6Subnet='2001:172:20:20::/80', MTU='1500'
INFO[0000] Creating container: client2
INFO[0000] Creating container: client1
INFO[0000] Creating container: leaf1
INFO[0000] Creating container: leaf4
INFO[0000] Creating container: leaf3
INFO[0000] Creating container: spine2
INFO[0000] Creating container: leaf2
INFO[0000] Creating container: spine1
INFO[0003] Restarting 'leaf4' node
INFO[0003] Restarting 'leaf2' node
INFO[0003] Restarting 'leaf3' node
INFO[0003] Restarting 'spine2' node
INFO[0003] Restarting 'leaf1' node
INFO[0003] Restarting 'spine1' node
ERRO[0003] failed to run postdeploy task for node spine1: Error response from daemon: page not found
ERRO[0003] failed to run postdeploy task for node leaf2: Error response from daemon: page not found
ERRO[0003] failed to run postdeploy task for node leaf1: Error response from daemon: page not found
ERRO[0003] failed to run postdeploy task for node spine2: Error response from daemon: page not found
ERRO[0003] failed to run postdeploy task for node leaf3: Error response from daemon: page not found
ERRO[0003] failed to run postdeploy task for node leaf4: Error response from daemon: page not found
INFO[0003] Creating virtual wire: leaf4:e3 <--> spine2:e5
INFO[0003] Creating virtual wire: leaf1:e3 <--> spine2:e2
INFO[0003] Creating virtual wire: leaf3:e10 <--> leaf4:e10
INFO[0003] Creating virtual wire: leaf3:e2 <--> spine1:e4
INFO[0003] Creating virtual wire: client1:eth1 <--> leaf1:e4
INFO[0003] Creating virtual wire: client2:eth2 <--> leaf4:e4
INFO[0003] Creating virtual wire: leaf3:e11 <--> leaf4:e11
INFO[0003] Creating virtual wire: leaf2:e3 <--> spine2:e3
INFO[0003] Creating virtual wire: leaf2:e2 <--> spine1:e3
INFO[0003] Creating virtual wire: client1:eth2 <--> leaf2:e4
INFO[0003] Creating virtual wire: leaf1:e2 <--> spine1:e2
INFO[0003] Creating virtual wire: leaf1:e10 <--> leaf2:e10
INFO[0003] Creating virtual wire: leaf4:e2 <--> spine1:e5
INFO[0003] Creating virtual wire: leaf3:e3 <--> spine2:e4
INFO[0003] Creating virtual wire: leaf1:e11 <--> leaf2:e11
INFO[0003] Creating virtual wire: client2:eth1 <--> leaf3:e4
INFO[0005] Writing /etc/hosts file
+---+--------------------------+--------------+---------------------------------+-------+-------+---------+----------------+----------------------+
| # |           Name           | Container ID |              Image              | Kind  | Group |  State  |  IPv4 Address  |     IPv6 Address     |
+---+--------------------------+--------------+---------------------------------+-------+-------+---------+----------------+----------------------+
| 1 | clab-ceos-clos01-client1 | 119fbee7fe5d | ghcr.io/hellt/network-multitool | linux |       | running | 172.20.20.3/24 | 2001:172:20:20::3/80 |
| 2 | clab-ceos-clos01-client2 | 3815f6e0754b | ghcr.io/hellt/network-multitool | linux |       | running | 172.20.20.6/24 | 2001:172:20:20::6/80 |
| 3 | clab-ceos-clos01-leaf1   | b631be404728 | ceos                            | ceos  |       | running | 172.20.20.5/24 | 2001:172:20:20::5/80 |
| 4 | clab-ceos-clos01-leaf2   | be4f6566dc70 | ceos                            | ceos  |       | running | 172.20.20.2/24 | 2001:172:20:20::2/80 |
| 5 | clab-ceos-clos01-leaf3   | b73e04ee805a | ceos                            | ceos  |       | running | 172.20.20.7/24 | 2001:172:20:20::7/80 |
| 6 | clab-ceos-clos01-leaf4   | 2fbb1e92f3d4 | ceos                            | ceos  |       | running | 172.20.20.9/24 | 2001:172:20:20::9/80 |
| 7 | clab-ceos-clos01-spine1  | 1e4b1588d3a2 | ceos                            | ceos  |       | running | 172.20.20.4/24 | 2001:172:20:20::4/80 |
| 8 | clab-ceos-clos01-spine2  | e5ca2159049a | ceos                            | ceos  |       | running | 172.20.20.8/24 | 2001:172:20:20::8/80 |
+---+--------------------------+--------------+---------------------------------+-------+-------+---------+----------------+----------------------+
dharmesh@dharmesh-ubuntu:~/clab-quickstart/clab-ceos-clos01$ docker -v
Docker version 20.10.5, build 55c4c88
dharmesh@dharmesh-ubuntu:~/clab-quickstart/clab-ceos-clos01$ uname -a
Linux dharmesh-ubuntu 5.8.0-48-generic #54~20.04.1-Ubuntu SMP Sat Mar 20 13:40:25 UTC 2021 x86_64 x86_64 x86_64 GNU/Linux
dharmesh@dharmesh-ubuntu:~/clab-quickstart/clab-ceos-clos01$

################# SSH to any hosts complain with no route to host for all cEOS, but to Linux it works#####
dharmesh@dharmesh-ubuntu:~/clab-quickstart/clab-ceos-clos01$ ping 172.20.20.3
PING 172.20.20.3 (172.20.20.3) 56(84) bytes of data.
64 bytes from 172.20.20.3: icmp_seq=1 ttl=64 time=0.295 ms
64 bytes from 172.20.20.3: icmp_seq=2 ttl=64 time=0.050 ms
^C
--- 172.20.20.3 ping statistics ---
2 packets transmitted, 2 received, 0% packet loss, time 1031ms
rtt min/avg/max/mdev = 0.050/0.172/0.295/0.122 ms
dharmesh@dharmesh-ubuntu:~/clab-quickstart/clab-ceos-clos01$ ping 172.20.20.5
PING 172.20.20.5 (172.20.20.5) 56(84) bytes of data.
From 172.20.20.1 icmp_seq=1 Destination Host Unreachable
From 172.20.20.1 icmp_seq=2 Destination Host Unreachable
From 172.20.20.1 icmp_seq=3 Destination Host Unreachable
^C
--- 172.20.20.5 ping statistics ---
5 packets transmitted, 0 received, +3 errors, 100% packet loss, time 4076ms
pipe 4
dharmesh@dharmesh-ubuntu:~/clab-quickstart/clab-ceos-clos01$

#########################console to cEOS ###############
dharmesh@dharmesh-ubuntu:~/clab-quickstart/clab-ceos-clos01$ sudo docker exec -it  clab-ceos-clos01-leaf1 Cli
leaf1>en
leaf1#sh ver
 cEOSLab
Hardware version:
Serial number:
Hardware MAC address: Not available
System MAC address: Not available

Software image version: 4.25.2F-20711308.4252F (engineering build)
Architecture: i686
Internal build version: 4.25.2F-20711308.4252F
Internal build ID: 6abe91ca-cb14-4038-96f6-f83dd0a346f5

cEOS tools version: 1.1
Kernel version: 5.8.0-48-generic

Uptime: 0 weeks, 0 days, 0 hours and 0 minutes
Total memory: 8123332 kB
Free memory: 2596984 kB

leaf1#sh int stat
Port       Name   Status       Vlan     Duplex Speed  Type         Flags Encapsulation
Ma0               connected    routed   a-full a-1G   10/100/1000

leaf1#

##################### Summary #########

Prob1) Do I need to create docker networks
# create docker networks
docker network create net1
docker network create net2
 
Prob2) cEOS doesnt show multiple interfaces

**Comment by hellt** [comment, marked as answer]
Hi,
try and change ceos interfaces from

* `e1` to `eth1`
* `e2` to `eth2` and so forth

**Reply by hellt** [reply]
and no, you do not need to create networks manually, containerlab takes care of this

**Reply by dharmbhai** [reply]
Hi @hellt 
Thanks for prompt reply. It seems to have partly worked, i.e replacing e2, e3 to eth2, eth3

I still get the errors, but the interfaces show up and topology is also built


But few problems still persist
1. Cannot SSH to only cEOS nodes on mgmt IP, Earlier, I wasnt able to ping, but now at least ping to cEOS works.
2. LLDP information of the built topology on one of the cEOS is incorrect


dharmesh@dharmesh-ubuntu:~/clab-quickstart/clab-ceos-clos01$ sudo containerlab deploy --reconfigure --topo ceos-clos01.clab.yml
INFO[0000] Parsing & checking topology file: ceos-clos01.clab.yml
INFO[0000] Destroying container lab: ceos-clos01
INFO[0000] Removing container entries from /etc/hosts file
INFO[0000] Deleting docker network 'clab'...
INFO[0000] Removing /home/dharmesh/clab-quickstart/clab-ceos-clos01/clab-ceos-clos01 directory...
INFO[0000] Creating lab directory: /home/dharmesh/clab-quickstart/clab-ceos-clos01/clab-ceos-clos01
INFO[0000] Creating docker network: Name='clab', IPv4Subnet='172.20.20.0/24', IPv6Subnet='2001:172:20:20::/80', MTU='1500'
INFO[0000] Creating container: leaf4
INFO[0000] Creating container: leaf3
INFO[0000] Creating container: client2
INFO[0000] Creating container: spine1
INFO[0000] Creating container: spine2
INFO[0000] Creating container: leaf1
INFO[0000] Creating container: client1
INFO[0000] Creating container: leaf2
INFO[0003] Restarting 'leaf3' node
INFO[0003] Restarting 'spine2' node
INFO[0003] Restarting 'leaf2' node
INFO[0003] Restarting 'leaf4' node
INFO[0003] Restarting 'leaf1' node
ERRO[0003] failed to run postdeploy task for node leaf2: Error response from daemon: page not found
ERRO[0003] failed to run postdeploy task for node spine2: Error response from daemon: page not found
ERRO[0003] failed to run postdeploy task for node leaf3: Error response from daemon: page not found
ERRO[0003] failed to run postdeploy task for node leaf1: Error response from daemon: page not found
INFO[0003] Restarting 'spine1' node
ERRO[0003] failed to run postdeploy task for node leaf4: Error response from daemon: page not found
ERRO[0003] failed to run postdeploy task for node spine1: Error response from daemon: page not found
INFO[0003] Creating virtual wire: leaf1:eth3 <--> spine2:eth2
INFO[0003] Creating virtual wire: leaf2:eth3 <--> spine2:eth3
INFO[0003] Creating virtual wire: leaf3:eth3 <--> spine2:eth4
INFO[0003] Creating virtual wire: leaf1:eth11 <--> leaf2:eth11
INFO[0003] Creating virtual wire: client2:eth2 <--> leaf4:eth4
INFO[0003] Creating virtual wire: leaf3:eth2 <--> spine1:eth4
INFO[0003] Creating virtual wire: leaf1:eth10 <--> leaf2:eth10
INFO[0003] Creating virtual wire: leaf3:eth10 <--> leaf4:eth10
INFO[0003] Creating virtual wire: leaf2:eth2 <--> spine1:eth3
INFO[0003] Creating virtual wire: leaf4:eth2 <--> spine1:eth5
INFO[0003] Creating virtual wire: client1:eth1 <--> leaf1:eth4
INFO[0003] Creating virtual wire: leaf4:eth3 <--> spine2:eth5
INFO[0003] Creating virtual wire: leaf3:eth11 <--> leaf4:eth11
INFO[0003] Creating virtual wire: client2:eth1 <--> leaf3:eth4
INFO[0003] Creating virtual wire: client1:eth2 <--> leaf2:eth4
INFO[0003] Creating virtual wire: leaf1:eth2 <--> spine1:eth2
INFO[0005] Writing /etc/hosts file
+---+--------------------------+--------------+---------------------------------+-------+-------+---------+----------------+----------------------+
| # |           Name           | Container ID |              Image              | Kind  | Group |  State  |  IPv4 Address  |     IPv6 Address     |
+---+--------------------------+--------------+---------------------------------+-------+-------+---------+----------------+----------------------+
| 1 | clab-ceos-clos01-client1 | 0d8c4034218e | ghcr.io/hellt/network-multitool | linux |       | running | 172.20.20.6/24 | 2001:172:20:20::6/80 |
| 2 | clab-ceos-clos01-client2 | a30add4068a4 | ghcr.io/hellt/network-multitool | linux |       | running | 172.20.20.7/24 | 2001:172:20:20::7/80 |
| 3 | clab-ceos-clos01-leaf1   | 799219f5dbe9 | ceos                            | ceos  |       | running | 172.20.20.3/24 | 2001:172:20:20::3/80 |
| 4 | clab-ceos-clos01-leaf2   | 32f609c02291 | ceos                            | ceos  |       | running | 172.20.20.9/24 | 2001:172:20:20::9/80 |
| 5 | clab-ceos-clos01-leaf3   | 8f19f7812df5 | ceos                            | ceos  |       | running | 172.20.20.8/24 | 2001:172:20:20::8/80 |
| 6 | clab-ceos-clos01-leaf4   | 3edf5d5a2ed0 | ceos                            | ceos  |       | running | 172.20.20.5/24 | 2001:172:20:20::5/80 |
| 7 | clab-ceos-clos01-spine1  | 96e947d8bc34 | ceos                            | ceos  |       | running | 172.20.20.4/24 | 2001:172:20:20::4/80 |
| 8 | clab-ceos-clos01-spine2  | 0ae330e5651e | ceos                            | ceos  |       | running | 172.20.20.2/24 | 2001:172:20:20::2/80 |
+---+--------------------------+--------------+---------------------------------+-------+-------+---------+----------------+----------------------+
dharmesh@dharmesh-ubuntu:~/clab-quickstart/clab-ceos-clos01$ ping 172.20.20.6
PING 172.20.20.6 (172.20.20.6) 56(84) bytes of data.
64 bytes from 172.20.20.6: icmp_seq=1 ttl=64 time=0.229 ms
64 bytes from 172.20.20.6: icmp_seq=2 ttl=64 time=0.046 ms
^C
--- 172.20.20.6 ping statistics ---
2 packets transmitted, 2 received, 0% packet loss, time 1014ms
rtt min/avg/max/mdev = 0.046/0.137/0.229/0.091 ms
dharmesh@dharmesh-ubuntu:~/clab-quickstart/clab-ceos-clos01$ ping 172.20.20.3
PING 172.20.20.3 (172.20.20.3) 56(84) bytes of data.
64 bytes from 172.20.20.3: icmp_seq=1 ttl=64 time=0.126 ms
64 bytes from 172.20.20.3: icmp_seq=2 ttl=64 time=0.071 ms
^C
--- 172.20.20.3 ping statistics ---
2 packets transmitted, 2 received, 0% packet loss, time 1024ms
rtt min/avg/max/mdev = 0.071/0.098/0.126/0.027 ms
dharmesh@dharmesh-ubuntu:~/clab-quickstart/clab-ceos-clos01$ ssh admin@clab-ceos-clos01-leaf1
ssh: connect to host clab-ceos-clos01-leaf1 port 22: No route to host
dharmesh@dharmesh-ubuntu:~/clab-quickstart/clab-ceos-clos01$ ssh admin@clab-ceos-clos01-leaf1
ssh: connect to host clab-ceos-clos01-leaf1 port 22: No route to host
dharmesh@dharmesh-ubuntu:~/clab-quickstart/clab-ceos-clos01$ sudo docker exec -it  clab-ceos-clos01-leaf1 Cli
leaf1>en
leaf1#sh int
Ethernet2 is up, line protocol is up (connected)
  Hardware is Ethernet, address is ea42.b49c.38c6 (bia ea42.b49c.38c6)
  Ethernet MTU 9214 bytes
  Full-duplex, Unconfigured, auto negotiation: off, uni-link: n/a
  Up 1 minute, 27 seconds
  Loopback Mode : None
  2 link status changes since last clear
  Last clearing of "show interface" counters never
  5 minutes input rate 0 bps (- with framing overhead), 0 packets/sec
  5 minutes output rate 0 bps (- with framing overhead), 0 packets/sec
     56 packets input, 6571 bytes
     Received 0 broadcasts, 56 multicast
     0 runts, 0 giants
     0 input errors, 0 CRC, 0 alignment, 0 symbol, 0 input discards
     0 PAUSE input
     0 packets output, 0 bytes
     Sent 0 broadcasts, 0 multicast
     0 output errors, 0 collisions
     0 late collision, 0 deferred, 0 output discards
     0 PAUSE output
Ethernet3 is up, line protocol is up (connected)
  Hardware is Ethernet, address is 76b8.c072.98fe (bia 76b8.c072.98fe)
  Ethernet MTU 9214 bytes
  Full-duplex, Unconfigured, auto negotiation: off, uni-link: n/a
  Up 1 minute, 27 seconds
  Loopback Mode : None
  2 link status changes since last clear
  Last clearing of "show interface" counters never
  5 minutes input rate 0 bps (- with framing overhead), 0 packets/sec
leaf1#sh int st
Port       Name   Status       Vlan     Duplex Speed  Type            Flags Encapsulation
Et2               connected    1        full   unconf EbraTestPhyPort
Et3               connected    1        full   unconf EbraTestPhyPort
Et4               connected    1        full   unconf EbraTestPhyPort
Et10              connected    1        full   unconf EbraTestPhyPort
Et11              connected    1        full   unconf EbraTestPhyPort
Ma0               connected    routed   a-full a-1G   10/100/1000

leaf1#sh lldp neigh
Last table change time   : 0:01:22 ago
Number of table inserts  : 10
Number of table deletes  : 0
Number of table drops    : 0
Number of table age-outs : 0

Port          Neighbor Device ID       Neighbor Port ID    TTL
---------- ------------------------ ---------------------- ---
Et2           spine1                   Ethernet2           120
Et3           spine2                   Ethernet2           120
Et10          leaf2                    Ethernet10          120
Et11          leaf2                    Ethernet11          120
Ma0           0000.0000.0000           Management0         120
Ma0           spine2                   Management0         120
Ma0           leaf4                    Management0         120
Ma0           spine1                   Management0         120
Ma0           leaf2                    Management0         120
Ma0           leaf3                    Management0         120

leaf1#


leaf1#sh lldp neigh
Last table change time   : 0:02:47 ago
Number of table inserts  : 10
Number of table deletes  : 1
Number of table drops    : 0
Number of table age-outs : 1

Port          Neighbor Device ID       Neighbor Port ID    TTL
---------- ------------------------ ---------------------- ---
Et2           spine1                   Ethernet2           120
Et3           spine2                   Ethernet2           120
Et10          leaf2                    Ethernet10          120
Et11          leaf2                    Ethernet11          120
Ma0           spine2                   Management0         120
Ma0           leaf4                    Management0         120
Ma0           spine1                   Management0         120
Ma0           leaf2                    Management0         120
Ma0           leaf3                    Management0         120

leaf1#exit
dharmesh@dharmesh-ubuntu:~/clab-quickstart/clab-ceos-clos01$ ssh admin@clab-ceos-clos01-leaf1
ssh: connect to host clab-ceos-clos01-leaf1 port 22: No route to host
dharmesh@dharmesh-ubuntu:~/clab-quickstart/clab-ceos-clos01$ ping clab-ceos-clos01-leaf1
PING clab-ceos-clos01-leaf1(clab-ceos-clos01-leaf1 (2001:172:20:20::3)) 56 data bytes
From dharmesh-ubuntu (2001:172:20:20::1) icmp_seq=1 Destination unreachable: Address unreachable
From dharmesh-ubuntu (2001:172:20:20::1) icmp_seq=2 Destination unreachable: Address unreachable
From dharmesh-ubuntu (2001:172:20:20::1) icmp_seq=3 Destination unreachable: Address unreachable
^C
--- clab-ceos-clos01-leaf1 ping statistics ---
5 packets transmitted, 0 received, +3 errors, 100% packet loss, time 4072ms

dharmesh@dharmesh-ubuntu:~/clab-quickstart/clab-ceos-clos01$ ping 172.20.20.3
PING 172.20.20.3 (172.20.20.3) 56(84) bytes of data.
From 172.20.20.1 icmp_seq=1 Destination Host Unreachable
From 172.20.20.1 icmp_seq=2 Destination Host Unreachable
From 172.20.20.1 icmp_seq=3 Destination Host Unreachable
^C
--- 172.20.20.3 ping statistics ---
6 packets transmitted, 0 received, +3 errors, 100% packet loss, time 5116ms
pipe 4
dharmesh@dharmesh-ubuntu:~/clab-quickstart/clab-ceos-clos01$

**Reply by dharmbhai** [reply]
Hi @hellt 
If I am not asking too much, my topology is standard CLOS with 2 SPine and 4 Leafs.
Perhaps my MacBook Pro based Ubuntu VM is starved of resources (I allocated Ubuntu VM 8GB RAM and 6 CPU)
May I please ask you to try my topology on your lab servers ? 
Perhaps you could identify the issues faster.
Again, I am bound by my own MacBookPro, else would have tested this on high end server too.

**Reply by dharmbhai** [reply]
Hi @hellt 
Some good news:
I did console to Leaf2 and there Interfaces and LLDP info is accurate, only the Leaf1 is showing inaccurate info
Similarly, Leaf3 is showing inaccurate info, and leaf4 is inaccurate
The pattern is not clear.

leaf2#sh int st
Port       Name   Status       Vlan     Duplex Speed  Type            Flags Encapsulation
Et2               connected    1        full   unconf EbraTestPhyPort
Et3               connected    1        full   unconf EbraTestPhyPort
Et4               connected    1        full   unconf EbraTestPhyPort
Et10              connected    1        full   unconf EbraTestPhyPort
Et11              connected    1        full   unconf EbraTestPhyPort
Ma0               connected    routed   a-full a-1G   10/100/1000

leaf2#sh lldp neig
Last table change time   : 0:07:40 ago
Number of table inserts  : 10
Number of table deletes  : 1
Number of table drops    : 0
Number of table age-outs : 1

Port          Neighbor Device ID       Neighbor Port ID    TTL
---------- ------------------------ ---------------------- ---
Et2           spine1                   Ethernet3           120
Et3           spine2                   Ethernet3           120
Et10          leaf1                    Ethernet10          120
Et11          leaf1                    Ethernet11          120
Ma0           leaf1                    Management0         120
Ma0           leaf4                    Management0         120
Ma0           spine2                   Management0         120
Ma0           spine1                   Management0         120
Ma0           leaf3                    Management0         120

leaf2#





leaf1#sh int sta
Port       Name   Status       Vlan     Duplex Speed  Type            Flags Encapsulation
Et2               connected    1        full   unconf EbraTestPhyPort
Et3               connected    1        full   unconf EbraTestPhyPort
Et4               connected    1        full   unconf EbraTestPhyPort
Et10              connected    1        full   unconf EbraTestPhyPort
Et11              connected    1        full   unconf EbraTestPhyPort
Ma0               connected    routed   a-full a-1G   10/100/1000

leaf1#sh lldp neigh
Last table change time   : 0:08:51 ago
Number of table inserts  : 10
Number of table deletes  : 1
Number of table drops    : 0
Number of table age-outs : 1

Port          Neighbor Device ID       Neighbor Port ID    TTL
---------- ------------------------ ---------------------- ---
Et2           spine1                   Ethernet2           120
Et3           spine2                   Ethernet2           120
Et10          leaf2                    Ethernet10          120
Et11          leaf2                    Ethernet11          120
Ma0           spine2                   Management0         120
Ma0           leaf4                    Management0         120
Ma0           spine1                   Management0         120
Ma0           leaf2                    Management0         120
Ma0           leaf3                    Management0         120

leaf1#


leaf3#sh int st
Port       Name   Status       Vlan     Duplex Speed  Type            Flags Encapsulation
Et2               connected    1        full   unconf EbraTestPhyPort
Et3               connected    1        full   unconf EbraTestPhyPort
Et4               connected    1        full   unconf EbraTestPhyPort
Et10              connected    1        full   unconf EbraTestPhyPort
Et11              connected    1        full   unconf EbraTestPhyPort
Ma0               connected    routed   a-full a-1G   10/100/1000

leaf3#sh lldp neig
Last table change time   : 0:09:26 ago
Number of table inserts  : 10
Number of table deletes  : 1
Number of table drops    : 0
Number of table age-outs : 1

Port          Neighbor Device ID       Neighbor Port ID    TTL
---------- ------------------------ ---------------------- ---
Et2           spine1                   Ethernet4           120
Et3           spine2                   Ethernet4           120
Et10          leaf4                    Ethernet10          120
Et11          leaf4                    Ethernet11          120
Ma0           leaf1                    Management0         120
Ma0           leaf4                    Management0         120
Ma0           spine2                   Management0         120
Ma0           spine1                   Management0         120
Ma0           leaf2                    Management0         120

leaf3#


leaf4#sh int sta
Port       Name   Status       Vlan     Duplex Speed  Type            Flags Encapsulation
Et2               connected    1        full   unconf EbraTestPhyPort
Et3               connected    1        full   unconf EbraTestPhyPort
Et4               connected    1        full   unconf EbraTestPhyPort
Et10              connected    1        full   unconf EbraTestPhyPort
Et11              connected    1        full   unconf EbraTestPhyPort
Ma0               connected    routed   a-full a-1G   10/100/1000

leaf4#sh lldp neig
Last table change time   : 0:10:24 ago
Number of table inserts  : 10
Number of table deletes  : 1
Number of table drops    : 0
Number of table age-outs : 1

Port          Neighbor Device ID       Neighbor Port ID    TTL
---------- ------------------------ ---------------------- ---
Et2           spine1                   Ethernet5           120
Et3           spine2                   Ethernet5           120
Et10          leaf3                    Ethernet10          120
Et11          leaf3                    Ethernet11          120
Ma0           leaf1                    Management0         120
Ma0           spine2                   Management0         120
Ma0           spine1                   Management0         120
Ma0           leaf2                    Management0         120
Ma0           leaf3                    Management0         120

leaf4#

**Reply by hellt** [reply]
I can confirm that the same errors appear on my machine using your topology
Will have a look

**Reply by hellt** [reply]
ok, funny, can you

1. destroy the lab first with `containerlab des -t <filename>`
2. remove the `-` from the lab name on line1 and make it like `ceosclos01` and retry?

**Reply by hellt** [reply]
@dharmbhai the fix for that issue is in #359 
for now you can just eliminate the dash in the lab' name, but in next release this will be fixed
Thanks for reporting

**Reply by dharmbhai** [reply]
Hi @hellt 
I confirm, after removing the - in the 1st line, everything works perfectly fine.
